packages/mdl-elevation/_variables.scss

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/variables";
+
+$mdl-elevation-umbra-color: rgba(black, .2);
+$mdl-elevation-penumbra-color: rgba(black, .14);
+$mdl-elevation-ambient-color: rgba(black, .12);
+
+$mdl-elevation-umbra-map: (
+  0: "0px 0px 0px 0px #{$mdl-elevation-umbra-color}",
```

Comment:
Don't know if you've considered this alternative, but you could move the appending of the umbra color to the `mdl-elevation` polyfill, thus avoiding having to repeat the variable name in all of these lines.

No big deal either way, though, as it doesn't affect the output, which is where we would have to worry about file size.